### PR TITLE
DevEx: Consolidate package.json files

### PR DESCRIPTION
**Pull request by ericsorenson**
In preparation for a consolidated `package.json`

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-api/src/cases/practitionerCaseAssociationLambda.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ const { handle } = require('../middleware/apiGatewayHelper');
  */
 exports.handler = event =>
   handle(event, async () => {
-    const {userId} = event.pathParameters || {};
-    const {caseId} = event.pathParameters || {};
+    const { userId } = event.pathParameters || {};
+    const { caseId } = event.pathParameters || {};
```
Comment (kkoskelin) [inline web-api/src/cases/practitionerCaseAssociationLambda.js]: These two statements *could* be combined.

File: web-api/src/cases/verifyCaseForUserLambda.js [hunk 2]
```diff
@@ -10,16 +10,18 @@ const { handle } = require('../middleware/apiGatewayHelper');
  */
 exports.handler = event =>
   handle(event, async () => {
-    const {userId} = event.pathParameters || {};
-    const {caseId} = event.pathParameters || {};
+    const { userId } = event.pathParameters || {};
+    const { caseId } = event.pathParameters || {};
```
Comment (kkoskelin) [inline web-api/src/cases/verifyCaseForUserLambda.js]: These could also be combined.

File: web-api/src/cases/verifyPendingCaseForUserLambda.js [hunk 3]
```diff
@@ -10,8 +10,8 @@ const { handle } = require('../middleware/apiGatewayHelper');
  */
 exports.handler = event =>
   handle(event, async () => {
-    const {userId} = event.pathParameters || {};
-    const {caseId} = event.pathParameters || {};
+    const { userId } = event.pathParameters || {};
+    const { caseId } = event.pathParameters || {};
```
Comment (kkoskelin) [inline web-api/src/cases/verifyPendingCaseForUserLambda.js]: combine?

File: web-api/src/documents/virusScanPdfLambda.js [hunk 4]
```diff
@@ -12,14 +12,16 @@ const {
  */
 exports.handler = event =>
   handle(event, async () => {
-    const {documentId} = event.pathParameters;
+    const { documentId } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/documents/virusScanPdfLambda.js]: Other lambdas in this changeset are using the construction ` ... = event.pathParameters || {}`.
Should this one also do this? Or is this unnecessary here (or everywhere)?

File: web-api/src/workitems/getDocumentQCBatchedForSectionLambda.js [hunk 5]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {section} = event.pathParameters;
+    const { section } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getDocumentQCBatchedForSectionLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getDocumentQCBatchedForUserLambda.js [hunk 6]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {userId} = event.pathParameters;
+    const { userId } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getDocumentQCBatchedForUserLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getDocumentQCInboxForSectionLambda.js [hunk 7]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {section} = event.pathParameters;
+    const { section } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getDocumentQCInboxForSectionLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getDocumentQCInboxForUserLambda.js [hunk 8]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {userId} = event.pathParameters;
+    const { userId } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getDocumentQCInboxForUserLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getDocumentQCServedForSectionLambda.js [hunk 9]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {section} = event.pathParameters;
+    const { section } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getDocumentQCServedForSectionLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getDocumentQCServedForUserLambda.js [hunk 10]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {userId} = event.pathParameters;
+    const { userId } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getDocumentQCServedForUserLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getInboxMessagesForSectionLambda.js [hunk 11]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {section} = event.pathParameters;
+    const { section } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getInboxMessagesForSectionLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getInboxMessagesForUserLambda.js [hunk 12]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {userId} = event.pathParameters;
+    const { userId } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getInboxMessagesForUserLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getSentMessagesForSectionLambda.js [hunk 13]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {section} = event.pathParameters;
+    const { section } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getSentMessagesForSectionLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/getSentMessagesForUserLambda.js [hunk 14]
```diff
@@ -11,7 +11,7 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {userId} = event.pathParameters;
+    const { userId } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/getSentMessagesForUserLambda.js]: Should this use `event.pathParameters || {}` ?

File: web-api/src/workitems/setWorkItemAsReadLambda.js [hunk 15]
```diff
@@ -11,13 +11,15 @@ const { handle } = require('../middleware/apiGatewayHelper');
 exports.handler = event =>
   handle(event, async () => {
     const user = getUserFromAuthHeader(event);
-    const {workItemId} = event.pathParameters;
+    const { workItemId } = event.pathParameters;
```
Comment (kkoskelin) [inline web-api/src/workitems/setWorkItemAsReadLambda.js]: Should this use `event.pathParameters || {}` ?

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Some inconsistencies which do not originate with your authorship, but I feel ought to be addressed.  Or tell me I'm nuts. 🔩

**Review by kkoskelin — APPROVED** [review]

**Review by wbyoko — APPROVED** [review]

File: web-api/src/cases/generateDocketRecordPdfLambda.js [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+const {
+  headers,
+  sendError,
+  sendOk,
+} = require('../middleware/apiGatewayHelper');
+const {
+  NotFoundError,
+  UnauthorizedError,
+} = require('../../../shared/src/errors/errors');
+const { getUserFromAuthHeader } = require('../middleware/apiGatewayHelper');
```
Comment (kkoskelin) [inline web-api/src/cases/generateDocketRecordPdfLambda.js]: Yow, could be combined with other import from same file.

File: cypress/integration/file-a-answer.spec.js [hunk 17]
```diff
@@ -0,0 +1,40 @@
+describe('Filing an Answer', function() {
```
Comment (kkoskelin) [inline cypress/integration/file-a-answer.spec.js]: I thought this file was deleted in a recent PR?

File: cypress/support/database.js [hunk 18]
```diff
@@ -5,37 +5,35 @@ AWS.config.accessKeyId = 'noop';
 AWS.config.secretAccessKey = 'noop';
 AWS.config.region = 'us-east-1';
 
-const data = require('../../../web-api/storage/fixtures/seed');
+const data = require('../../web-api/storage/fixtures/seed');
```
Comment (kkoskelin) [inline cypress/support/database.js:8]: a cypress screenshot is being checked in, but that folder should probably be ignored. (I couldn't add this comment to it but it's at https://github.com/flexion/ef-cms/pull/2488/files?short_path=58060ad#diff-58060ada3e6606b78cda52e2d0325bc8 )

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**